packages/examples/packages/send-flow/src/test-utils/ constants.ts

```diff
@@ -0,0 +1,36 @@
+import type { CaipAssetType, CaipChainId } from '@metamask/utils';
+
+export const MOCK_ACCOUNTS = [
```

Comment:
Maybe we should export some helper functions to make creating these mocks easier?

Reply:
But rather than having an empty option by default I've changed it to include some default accounts in the simulation.